src/test/java/com/enterprise/eventmanagementsystem/EventmanagementsystemApplicationTests.java

```diff
@@ -10,69 +11,30 @@
 import static org.mockito.Mockito.verify;
 
 import java.text.ParseException;
-import java.text.SimpleDateFormat;
-
 
 @SpringBootTest
-class EventmanagementsystemApplicationTests {
-
-
+class EventManagementSystemApplicationTests {
 
     @MockBean
     private Event event= new Event();
     @MockBean
     private EventRepository eventRepository;
 
-    @Test
-    void contextLoads() {
-    }
     @Test
     void addNewEvent_displayNewEvent()throws ParseException {
-        givenNewEventIsAvailable();
-        whenCreatingNewEvent();
```

Comment:
I'd fix these instead of remove them.  BDD is "Given When Then", not just "Then".  The Given and When are the setup for what gets tested in the Then.